cockroachcloud/export-logs.md

```diff
@@ -173,21 +173,21 @@ Perform the following steps to enable log export from your {{ site.data.products
             {% include_cached copy-clipboard.html %}
             ~~~json
             {
-             type: AWS_CLOUDWATCH,
-             log_name: "default",
-             auth_principal: "{role_arn}",
-             redact: true,
-             region: "",
-             groups: [
+             "type": "AWS_CLOUDWATCH",
+             "log_name": "default",
```

Comment:
let's change this to `crdb-default` instead of `default` because the log group in their AWS account won't have any CRDB context otherwise.